alexa-news.php

```diff
@@ -32,7 +32,47 @@ public function news_request( $event ) {
 
 		switch ( $intent ) {
 			case 'Latest':
-				$result = $this->endpoint_content();
+				$term_slot = strtolower( sanitize_text_field( $request->getSlot( 'TermName' ) ) );
+
+				$args = [
+					'post_type'      => alexawp_news_post_types(),
+					'posts_per_page' => max( 1, min( 100, count( $this->placement ) ) ),
```

Comment:
Only because it's the highest allowed on WordPress VIP. It can be any number, really. The only point is to try to always fetch the same number of posts as we have placement phrases without accidentally creating a large query.